app/views/users/show.html.erb

```diff
@@ -1,161 +1,237 @@
-      <% if current_user&.id == @user.id %>
-        <div>
+        <% end %>
+        <% if current_user&.id == @user.id %>
           <%= link_to qr_login_code_path, class: 'button is-outlined is-small' do %>
             <i class="fas fa-mobile-alt"></i> Mobile Sign In
           <% end %>
+        <% end %>
+      </div>
+
+      <h2 class="user-profile-heading">Posts</h2>
+      <% if @user.posts.size == 0 %>
+        <p><%= @user.username %> hasn't posted anything yet.</p>
+      <% else %>
+        <div class="item-list user-profile-post-list">
+          <% @user.posts.undeleted.order(score: :desc).first(15).each do |a| %>
+            <%= render 'posts/type_agnostic', post: a, show_type_tag: true, show_category_tag: true %>
+          <% end %>
         </div>
+        <%= link_to user_posts_path(@user), class: "button is-muted" do %>
+          See all &raquo;
+        <% end %>
       <% end %>
     </div>
   </div>
-</div>
+  <div class="grid--cell is-3-lg is-12">
+    <div class="widget is-tertiary">
+      <div class="widget--body user-profile--image">
+        <img alt="user avatar" src="<%= avatar_url(@user, 512) %>" />
+      </div>
+      <% if @user.staff? %>
+      <div class="widget--body h-ta-center" title="This user is an official Codidact Staff member.">
+        <i class="fas fa-users-cog"></i> Codidact Staff
+      </div>
+      <% elsif @user.is_moderator || @user.is_admin %>
+      <div class="widget--body h-ta-center" title="This user is a moderator of the <%= SiteSetting['SiteName'] %> community.">
+        <i class="fas fa-shield-alt"></i> Moderator
+      </div>
+      <% end %>
+    </div>
 
+    <% unless @abilities.empty? %>
+      <h2>Earned Abilities</h2>
 
-<h2 class="user-profile-heading">Posts</h2>
-<% if @user.posts.size == 0 %>
-  <em class="text-muted"><%= @user.username %> hasn't posted anything yet.</em>
-<% else %>
-  <div class="item-list">
-    <% @user.posts.undeleted.order(score: :desc).first(5).each do |a| %>
-      <%= render 'posts/type_agnostic', post: a, show_type_tag: true, show_category_tag: true %>
+      <div class="widget">
+        <% @abilities.each do |a| %>
+          <div class="widget--body">
+            <i class="fa-fw <%= a.icon %>"></i>
+            <a href="<%= ability_url(a.internal_id) %>">
+              <%= a.name %>
+            </a>
+          </div>
+        <% end %>
+      </div>
     <% end %>
-  </div>
-  <%= link_to user_posts_path(@user), class: "button is-muted" do %>
-    See all &raquo;
-  <% end %>
-<% end %>
 
-<%
-  votes_by_type = @user.votes.group(:vote_type).count(:vote_type)
-  votes_by_post_type = @user.votes.joins(:post).group(Arel.sql('posts.post_type_id')).count(Arel.sql('posts.post_type_id'))
-%>
+    <h2>Statistics</h2>
+    <%
+      posts_by_post_type = @user.posts.undeleted.group(Arel.sql('posts.post_type_id')).count(Arel.sql('posts.post_type_id'))
+    %>
+    <table class="table is-full-width">
+      <tr>
+        <th colspan=3>
```

Comment:
Always quote HTML tag attributes.

Also applies to other occurrences in this file.

```suggestion
        <th colspan="3">
```